js/src/common/components/LoadingIndicator.tsx

```diff
@@ -40,21 +42,30 @@ export interface LoadingIndicatorAttrs extends ComponentAttrs {
  *
  * - `containerClassName` Class name(s) to apply to the indicator's parent
  * - `className` Class name(s) to apply to the indicator itself
- * - `size` Size of the loading indicator
+ * - `size` Size of the loading indicator (small, medium or large)
```

Comment:
You changed `tiny` to `small` but the CSS still targets `tiny`, any reason for the change ?

Reply:
To me, small medium and large sounds more natural than tiny medium and large.